frontend/src/components/Authentication/Login.js

```diff
@@ -13,18 +13,37 @@ const Login = () => {
   const [show, setShow] = useState(false);
   const handleClick = () => setShow(!show);
   const toast = useToast();
-  const [email, setEmail] = useState();
-  const [password, setPassword] = useState();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
   const [loading, setLoading] = useState(false);
   const { setUser } = ChatState();
 
   const navigate = useNavigate();
 
+  const isValidEmail = (email) => {
+    // Regular expression for validating an Email
+    const regex = /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/;
```

Comment:
will this only accept emails with prefix .gmail???

Reply:
what happens when use gives another domain

Reply:
yes it will, It's the common format for any domain
emails are like something@something.something and we need this format only. therefore, it's workable.